travis-ci/vmtest/prepare_selftests.sh

```diff
@@ -4,10 +4,14 @@ set -eu
 
 source $(cd $(dirname $0) && pwd)/helpers.sh
 
-REPO_PATH=$1
+REPO_PATH=${1:-}
 
-${VMTEST_ROOT}/checkout_latest_kernel.sh ${REPO_PATH}
-cd ${REPO_PATH}
+
+
+if [[ ! -z "$REPO_PATH" ]]; then
+  ${VMTEST_ROOT}/checkout_latest_kernel.sh ${REPO_PATH}
+  cd ${REPO_PATH}
```

Comment:
please use tabs consistently in this file (or we should just normalize one way or another everywhere)